addons/core/intercomConfig/config.cpp

```diff
@@ -15,19 +15,19 @@ class CfgPatches {
 
 #define IntercomMacro class ACE_SelfActions : ACE_SelfActions { \
     class TFAR_IntercomChannel { \
-        displayName = "IntercomChannel"; \
+        displayName = "$STR_TFAR_Intercom_ACESelfAction_Name"; \
```

Comment:
Didn't notice this before. Your names don't match the CBA/ACE macro style.
They use STR_Addon_Module_name
The module here is Core
so STR_TFAR_Core_Intercom_ACESelfAction_Name
See
https://github.com/acemod/ACE3/blob/master/addons/main/config.cpp#L901
https://github.com/acemod/ACE3/blob/master/addons/main/stringtable.xml#L19

The 1.0 update is also a move to ACE/CBA standards. So we should use the macros where appropriate. The macros can be done later. But the strings should be ready for them. Atleast the new strings that get added. The old ones from 0.9.12 are still not conformant I think so they need to be changed some day but not in this PR.